src/ODM/MongoDB/Types/CurrencyType.php

```diff
@@ -2,10 +2,10 @@
 
 namespace ZFBrasil\DoctrineMoneyModule\ODM\MongoDB\Types;
 
-use Doctrine\ODM\MongoDB\Types\StringType;
+use Doctrine\ODM\MongoDB\Types\Type;
 use Money\Currency;
 
-class CurrencyType extends StringType
+class CurrencyType extends Type
```

Comment:
Can you revert this one? We still can inherit `convertToDatabaseValue()` and `closureToMongo()` from `StringType` since `Currency` implements `__toString()` method.